stix_shifter_modules/sentinelone/tests/stix_translation/test_sentinelone_json_to_stix.py

```diff
@@ -0,0 +1,3126 @@
+import unittest
+from stix_shifter_utils.stix_translation.src.json_to_stix import json_to_stix_translator
+from stix_shifter_modules.sentinelone.entry_point import EntryPoint
+from stix_shifter_utils.stix_translation.src.utils.transformer_utils import get_module_transformers
+
+MODULE = "sentinelone"
+entry_point = EntryPoint()
+map_data = entry_point.get_results_translator().map_data
+data_source = {
+    "type": "identity",
+    "id": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+    "name": "sentinelone",
+    "identity_class": "events"
+}
+options = {}
+
+
+class TestSentineloneResultsToStix(unittest.TestCase):
+    """
+    class to perform unit test case for sentinelone translate results
+    """
+
+    @staticmethod
+    def get_first(itr, constraint):
+        """
+        return the obj in the itr if constraint is true
+        """
+        return next(
+            (obj for obj in itr if constraint(obj)),
+            None
+        )
+
+    @staticmethod
+    def get_first_of_type(itr, typ):
+        """
+        to check whether the object belongs to respective stix object
+        """
+        return TestSentineloneResultsToStix.get_first(itr,
+                                                      lambda o: isinstance(o, dict) and
+                                                                o.get('type') == typ)
+
+    def test_common_prop(self):
+        """to test common stix object properties"""
+        data = {"activeContentFileId": None,
```

Comment:
If this data is the same across all tests, it would be more readable to store this in a constant to be used for each test.

Reply:
done the changes as suggested, some common datafield is kept in constant and reused in different function.